src/ucp/proto/proto_debug.c

```diff
@@ -445,28 +429,15 @@ void ucp_proto_config_info_str(ucp_worker_h worker,
                               proto_attr.desc, proto_attr.config);
     ucs_string_buffer_rtrim(strb, NULL);
 
-    /* Find protocol selection root */
-    proto_select = ucp_proto_select_get(worker, proto_config->ep_cfg_index,
-                                        proto_config->rkey_cfg_index,
-                                        &new_key_cfg_index);
-    if (proto_select == NULL) {
-        return;
-    }
-
-    /* Emulate protocol selection process */
-    ucs_assert(new_key_cfg_index == proto_config->rkey_cfg_index);
-    select_elem = ucp_proto_select_lookup_slow(worker, proto_select, 1,
-                                               proto_config->ep_cfg_index,
-                                               proto_config->rkey_cfg_index,
-                                               &proto_config->select_param);
-    if (select_elem == NULL) {
+    /* Find the relevant performance range */
+    range = ucp_proto_flat_perf_find_lb(proto_config->init_elem->flat_perf,
+                                        msg_length);
+    if (range == NULL || range->start > msg_length) {
+        ucs_string_buffer_appendf(strb, " - not available");
         return;
     }
 
-    /* Find the relevant performance range */
-    range     = ucp_proto_perf_range_search(select_elem, msg_length);
-    bandwidth = ucp_proto_select_calc_bandwidth(&proto_config->select_param,
-                                                range, msg_length);
+    bandwidth = msg_length / ucs_linear_func_apply(range->value, msg_length);
```

Comment:
I found couple of places where we are doing same operations to calculate BW and it seems like in all of these places can divide result by 0. But since all these places use this result for printing, and result of 0 division would be INF I think that is OK ("inf" would be printed, which is basically true in that case. see https://godbolt.org/z/66cqTo7YT).

If we want to consider infinite BW as an issue, we can add assert to catch such cases. WDYT?